views/js/qtiCreator/widgets/static/text/states/Active.js

```diff
@@ -31,50 +34,41 @@ define([
 
         $editableContainer.attr('data-html-editable-container', true);
 
-        if (!htmlEditor.hasEditor($editableContainer)) {
-            htmlEditor.buildEditor($editableContainer, {
-                change: function (data) {
-                    changeCallback.call(this, data);
-                    if (!data) {
-                        widget.$form.find('[name="textBlockCssClass"]').val('');
-                    }
-                },
-                blur: function () {
-                    widget.changeState('sleep');
-                },
-                data: {
-                    widget: widget,
-                    container: container
+        const defaultEditorOptions = {
+            change: function (data) {
+                changeCallback.call(this, data);
+                if (!data) {
+                    widget.$form.find('[name="textBlockCssClass"]').val('');
                 }
-            });
-        }
-    };
+            },
+            blur: function () {
+                widget.changeState('sleep');
+            },
+            data: {
+                widget: widget,
+                container: container
+            }
+        };
 
-    TextActive.prototype.destroyEditor = function () {
-        htmlEditor.destroyEditor(this.widget.$container);
-    };
+        const getEditorOptions = function () {
+            const editorOptions = {};
+            const removePlugins = [];
 
-    TextActive.prototype.initForm = function () {
-        const widget = this.widget,
-            $form = widget.$form,
-            $wrap = widget.$container.find(`.${wrapperCls}`),
-            blockCls = itemScrollingMethods.cutScrollClasses($wrap.attr('class') || ''),
-            isScrolling = itemScrollingMethods.isScrolling($wrap),
-            selectedHeight = itemScrollingMethods.selectedHeight($wrap);
-
-        $form.html(
-            formTpl({
-                textBlockCssClass: blockCls.replace(wrapperCls, '').trim(),
-                scrolling: isScrolling,
-                scrollingHeights: itemScrollingMethods.options()
-            })
-        );
+            if (isHiddenPlugin('taotooltip')) {
```

Comment:
question(optional): is it possible to iterate over the list of plugins? Since you've unlocked the possibility to [check the visibility for every plugin](https://github.com/oat-sa/extension-tao-itemqti/pull/2102/files#diff-478eea9160571680a8eb23aae710c2c271b62c7925a9df15037bea47c0c346f1R15) it would be wonderful to remove all hidden plugin :)

Reply:
That is nice idea, I've managed to get all registerered ckeditor plugins to loop through, so now we can disable any of them